x-pack/plugins/maps/public/classes/layers/vector_tile_layer/vector_tile_layer.js

```diff
@@ -56,7 +56,7 @@ export class VectorTileLayer extends TileLayer {
 
     const requestToken = Symbol(`layer-source-refresh:${this.getId()} - source`);
     try {
-      startLoading(SOURCE_DATA_REQUEST_ID, requestToken, dataFilters);
+      startLoading(SOURCE_DATA_REQUEST_ID, requestToken, { dataFilters, ...nextMeta });
```

Comment:
Do you even need to add dataFilters to meta parameter? They are never read anywhere so I think its best to remove then from meta since they are not used by this layer